src/umi_tools/umi_tools_extract/config.vsh.yaml

```diff
@@ -0,0 +1,180 @@
+
+name: umi_tools_extract
+namespace: umi_tools
+description: |
+  Flexible removal of UMI sequences from fastq reads.
+  UMIs are removed and appended to the read name. Any other barcode, for example a library barcode,
+  is left on the read. Can also filter reads by quality or against a whitelist.
+keywords: [ extract, umi-tools, umi, fastq ]
+links:
+  homepage: https://umi-tools.readthedocs.io/en/latest/
+  documentation: https://umi-tools.readthedocs.io/en/latest/reference/extract.html,
+                 https://umi-tools.readthedocs.io/en/latest/common_options.html#common-options
```

Comment:
Could you simplify this to just one link?

If you want the capability of specifying a list of links, please create an issue in `viash-io/viash`!